File: RAI/EjemploBanco/JavaImplementation/Hipotecario.java
public class Hipotecario extends Credito{
    private String propiedad;

    public Hipotecario(int id, double interes, int monto, int cuotas, String propiedad){
        super(id, interes, monto, cuotas);
        this.propiedad = propiedad;
    }
    
    public String toString(){
        return super.toString() + "Propiedad: " + this.propiedad;
    }
}

File: RAI/EjemploBanco/python_implementation/Hipotecario.py
from Credito import Credito

class Hipotecario(Credito):
    
    def __init__(self, identificador, interes, monto, cuotas, propiedad):
        super().__init__(identificador, interes, monto, cuotas)
        self.propiedad = propiedad

    def __str__(self):
        return super().__str__() + " Propiedad: " + self.propiedad